tests/functional/test_conversions.py

```diff
@@ -223,8 +222,19 @@ def test_mtx(self, mock_upload_method):
                 mtx_cells[row["cellkey"]] = row
 
             mtx_genes = collections.OrderedDict()
-            for row in csv.DictReader(io.StringIO(gzip.GzipFile(fileobj=io.BytesIO(
-                    mtx_output.read("test.mtx/genes.tsv.gz"))).read().decode()), delimiter='\t'):
+            for row in csv.DictReader(
+                    io.StringIO(gzip.GzipFile(fileobj=io.BytesIO(
+                    mtx_output.read("test.mtx/features.tsv.gz"))).read().decode()),
+                    delimiter='\t',
+                    fieldnames=["featurekey",
```

Comment:
Required since `features.tsv.gz` per specification does not include headers. Will this be confusing to users given we're removing `genes.tsv.gz`?